app/views/publish/courses/_basic_details_tab.html.erb

```diff
@@ -139,7 +139,12 @@
 
       <% summary_list.row(html_attributes: { data: { qa: "course__start_date" } }) do |row| %>
         <% row.key { "Course starts" } %>
-        <% row.value { l(course.start_date&.to_date, format: :short) } %>
+        <% row.value do %>
+          <ul class="govuk-list">
+            <li><%= l(course.start_date&.to_date, format: :short) %></li>
+            <li>Academic year <%= course.cycle_range %></li>
```

Comment:
I think academic year is a brand new thing we are introducing. It would be a year earlier than the `cycle_range`. So for the prototype, in the current cycle it's `Academic year 2021 to 2022` and for next it's `Academic year 2022 to 2023`. I think we want to match that

Reply:
Sure, the info I had on the ticket
![image](https://user-images.githubusercontent.com/4527528/176684790-9131b70f-d8c7-4e73-a7f4-3951515c577e.png) seemed to work with the cycle range. Are we to introduce a new attribute for the course then?

Reply:
We have 
```
def cycle_range
  "#{course.recruitment_cycle.year} to #{course.recruitment_cycle.year.to_i + 1}"
end
```
So
```
def academic_year
  "#{course.recruitment_cycle.year.to_i - 1} to #{course.recruitment_cycle.year.to_i }"
end
```

Reply:
Yeah, I think that looks about right now, the hint text looks right there too

Reply:
Does it work if you drop the `to_i` on the second part? `to #{course.recruitment_cycle.year.to_i }`